spec/services/hyrax/custom_queries/navigators/find_files_spec.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.describe Hyrax::CustomQueries::Navigators::FindFiles do
+  let(:query_service) { Valkyrie::MetadataAdapter.find(:test_adapter).query_service }
+
+  describe '#find_files' do
+    subject { query_service.custom_queries.find_files(resource: fileset) }
```

Comment:
I'd prefer if we avoid this style. It seems like unnecessary indirection that makes tests hard to read. Could we make `subject` the unit under test?

`subject(:query_handler) { described_class.new(query_service: query_service) }`

Reply:
DONE - matches style of PR #4078